lib/log.rb

```diff
@@ -158,9 +158,8 @@ def print(msg)
 
     # supports embedding content similar with same semantics as Cucumber
     def embed(src, mime_type, label)
-      if self.class.runtime.respond_to? :embed
-        info "embedding #{label.inspect}"
-        self.class.runtime.embed src, mime_type, label
+      if self.class.runtime.respond_to? :attach
+        self.class.runtime.attach src, mime_type
```

Comment:
Could you keep this line `info "embedding #{label.inspect}"` because it will be really helpful finding the correct screenshot when too many screenshot is saved

Reply:
The label here is empty ATM. Keep it does not help much.

Reply:
Move/add them in https://github.com/openshift/verification-tests/pull/2025/files#diff-fcf1e86594bb0416d6eecb2efacfc0de19c92dba55fdcf6848142a697e211657R41